managed/services/telemetry/datasource_envvars.go

```diff
@@ -20,7 +20,7 @@ import (
 	"context"
 	"os"
 
-	pmmv1 "github.com/percona-platform/saas/gen/telemetry/events/pmm"
+	genericv1 "github.com/percona/saas/gen/telemetry/generic"
```

Comment:
I would call it `telemetryv1`, since `generic` doesn't tell much when you see it in the code.

Reply:
Both (genericv1 and reporter) renamed to telemetryv1.